src/EventStore.Core/Services/Transport/Common/Position.cs

```diff
@@ -150,7 +150,7 @@ public static Position FromInt64(long commitPosition, long preparePosition)
 		/// A <see cref="T:System.String"/> containing a fully qualified type name.
 		/// </returns>
 		/// <filterpriority>2</filterpriority>
-		public override string ToString() => $"C:{CommitPosition}/P:{PreparePosition}";
+		public override string ToString() => this == End ? "End" : $"C:{CommitPosition}/P:{PreparePosition}";
```

Comment:
@YoEight hey, thanks for the review.

i've just double checked it by looking for usages of `Position.End`. it seems it's used only by the following enumerators:
- $all/filtered $all reads
- $all/filtered $all subscriptions
- persistent subscriptions (creating & updating)

and it appears that the $all filtered/reads and persistent subscriptions enumerators don't log anything at the moment, so it should be a safe change. in the worst case, instead of logging `C: <ulong.maxValue>/P:<ulong.MaxValue>`, it would log `End` which should be ok.

Reply:
I was also concerned about external tools that might use it, like some Prometheus exporters. Granted it's out of our scope.